### DISCUSSION TITLE
[Question] Import Isaac Sim pancakes in VScode

### DISCUSSION THREAD

**Discussion by relifeto18** [isaac-sim/IsaacLab#1045] [Q&A]
### Question

I've followed the [tutorial](https://isaac-sim.github.io/IsaacLab/source/setup/installation/pip_installation.html#installation-using-isaac-sim-pip) to install Isaac Lab and Isaac SIm using pip (newest version) in conda environment. 

When running `./isaaclab.sh --install`, a warning showed up:

![image](https://github.com/user-attachments/assets/9c7776db-af25-4a8c-ba40-0eccd40664f4)

I mainly use Issac Sim for my project. When I write code in VScode under my virtual environment, the editor seems unable to find the packages. 

![image](https://github.com/user-attachments/assets/a90fcc1e-96f1-4f89-a8f1-a6d656c37acd)

I can run my code in the terminal (the installation should be correct) but it's super annoying if I want to check a function's definition when writing the code. Can you tell me how to fix this problem?

**Comment by glvov-bdai** [comment]
Hi,

Try creating a new vscode workspace, and open ONLY the isaac lab folder. 

Do 

CTRL + SHIFT + P (opens command pallete) -> Tasks: Run Task -> setup python env

Paths should now resolve

https://isaac-sim.github.io/IsaacLab/source/overview/developer-guide/vs_code.html

**Reply by relifeto18** [reply]
Hello, 

Thanks for getting back to me. What I'm trying to do is to build a workspace outside the IsaacLab folder. I believe following the tutorial (as I did) does not solve this problem. And this is more like Isaac Sim side.

**Reply by glvov-bdai** [reply]
You can try doing what I suggested, and then adding the folder to the VS code workspace after you setup the python environment,

Alternatively, you can also try placing your own folder into the isaaclab folder, and just add it to the gitignore for isaac lab
